docs/management.md

```diff
@@ -293,42 +299,63 @@ rabbitmqctl set_user_tags full_access administrator
+### Basic configuration {#basic-configuration}
 
-To redirect users to the UAA server to authenticate, use the following configuration:
+Given the following configuration of the [OAuth 2.0 plugin](./oauth2):
+
+```ini
+auth_oauth2.resource_server_id = new_resource_server_id
+auth_oauth2.issuer = https://my-oauth2-provider.com/realm/rabbitmq
+```
+
+Apply the following configuration to enable OAuth 2.0 authentication:
 
 ```ini
 management.oauth_enabled = true
 management.oauth_client_id = rabbit_user_client
-management.oauth_client_secret = rabbit_user_client
-management.oauth_provider_url = https://my-uaa-server-host:8443/uaa
 management.oauth_scopes = openid profile rabbitmq.*
 ```
 
-> **Important**: UAA supports regular expression in scopes, e.g. `rabbitmq.*`. The above configuration
-assumes that the `resource_server_id` configured in the oauth2 backend matches the value `rabbitmq`.
+- `oauth_enabled` is mandatory and it enables OAuth 2.0 authentication provided we have OAuth 2.0 plugin configured too.
+- `oauth_client_id` is mandatory and it is the OAuth Client Id associated to this RabbitMQ cluster in the OAuth Provider and it is used to request a token on behalf of the user
+- `oauth_scopes` most of the time this setting is mandatory. It is a list of space-separated strings indicating which permissions the application is requesting. Most OAuth providers will only issue tokens with the scopes requested during the user authentication. RabbitMQ sends this field along with its `oauth_client_id` during the user authentication. However, there are OAuth providers who may automatically grant scopes associated to the `oauth_client_id`. Only in this latter case, we do not need to specify the `oauth_scopes` field.
+
+If `management.oauth_scopes` is not set, RabbitMQ defaults to `openid profile`.
 
-> **Important**: Since RabbitMQ 3.10, RabbitMQ uses `authorization_code` grant type. `implicit` flow is deprecated.
+Depending on the Authorization server, you may use regular expression in scopes, e.g. `[resource_server_id].*`, or
+instead you have to explicitly ask for them, e.g.: Below `rabbitmq` value is used as the resource_server_id.
+  - `rabbitmq.tag:administrator`
+  - `rabbitmq.read:*/*/*`
 
-> **Important**: `management.oauth_client_secret` is an optional setting. UAA 75.21.0 and earlier versions require `oauth_client_secret` regardless if the oauth client is configured as confidental.
+Since RabbitMQ 3.13, it is no longer necessary to set the url of the OAuth 2.0 provider in `management.oauth_provider_url` setting. This URL is now read from the `auth_oauth2.issuer` setting. The management UI uses that URL to download the OpenID Provider configuration via the URL `auth_oauth2.issuer` followed by the path `/.well-known/openid-configuration`. If management UI cannot download the OpenID Provider configuration, it displays a warning message and users cannot login via OAuth 2.0.
+
```

Comment:
Is the "Since RabbitMQ 3.13......" information the next step in this "Basic Configuration" procedure then? we are telling users about another field: `management.oauth_provider_url` (I know they don't have to do anything with it) but this is the first time it is mentioned, it is not here in the configuration they are applying: 

```
management.oauth_enabled = true
management.oauth_client_id = rabbit_user_client
management.oauth_scopes = openid profile rabbitmq.*
```
So users (new users anyway) will have no context/history of using this field - I guess a user new to RabbitMQ and using RabbitMQ 3.13 for the first time won't care as they don't have to set this field but I guess I would provide more context here about when setting (and not setting as the case is now) this field happens in the "Basic Configuration" steps

Reply:
Excellent feedback ! thanks @pstack2021